hf-example/index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+        <link rel="stylesheet" type="text/css" href="https://api.pryv.com/style/pryv.min.css">
+        <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400">
+        <link href="style.css" rel="stylesheet" type="text/css" />
+        <title>Pryv HF</title>
+        <script src="https://api.pryv.com/lib-js/pryv.js"></script>
+        <script src="https://threejs.org/build/three.js"></script>
+        <script src="script.js"></script>
+    </head>
+
+    <body>
+        <div class="container">
+            <h1>Pryv.io collect and visualize HF Data</h1>
+
+            <div class="card">
+                <div class="card-body">
+                    <h2 class="card-title">Welcome</h2>
+                    <br />
+                    <p>Click on fetch -> login -> mouse tracker</p>
+
+                </div>
+            </div>
+
+            <div class="card" id="service">
+                <div class="card-body">
+                    <button class="collapsible">
+            Service information
+          </button>
+                    <div class="service_info">
+                        <br> Enter the <a href="https://api.pryv.com/reference/#service-info">service information URL</a> to fetch the required information to customize your welcome message and fetch the URL for the authentication process.
```

Comment:
Enter the <a href="https://api.pryv.com/reference/#service-info">service information URL</a>  for the authentication process.

Reply:
Fixed